packages/suite/src/views/settings/SettingsCoins/SettingsCoins.tsx

```diff
@@ -62,13 +70,20 @@ export const SettingsCoins = () => {
             )}
             {showFirmwareTypeBanner && <FirmwareTypeSuggestion />}
             <SettingsSection title={<Translation id="TR_COINS" />} icon="COIN">
-                <SectionItem ref={anchorRefCrypto} shouldHighlight={shouldHighlightCrypto}>
-                    <CoinGroup
-                        networks={mainnets}
-                        onToggle={setEnabled}
-                        selectedNetworks={enabledNetworks}
-                    />
-                </SectionItem>
+                <div>
+                    <SectionItem ref={anchorRefCrypto} shouldHighlight={shouldHighlightCrypto}>
+                        <CoinGroup
+                            networks={mainnets}
+                            onToggle={setEnabled}
+                            selectedNetworks={enabledNetworks}
+                        />
+                    </SectionItem>
+                    {isDiscoveryButtonVisible && (
+                        <Button onClick={startDiscovery}>
+                            <Translation id="TR_DISCOVERY_NEW_COINS" />
+                        </Button>
+                    )}
```

Comment:
No need for a separator like in the design but definitely a `16px` margin-top` is required here 🙏